source/features/more-dropdown.tsx

```diff
@@ -53,18 +61,56 @@ async function init(): Promise<void> {
 	const compareUrl = `/${repoUrl}/compare/${reference}`;
 	const commitsUrl = `/${repoUrl}/commits/${reference}`;
 	const dependenciesUrl = `/${repoUrl}/network/dependencies`;
+	const branchesUrl = `/${repoUrl}/branches`;
 
 	const nav = select('.js-responsive-underlinenav .UnderlineNav-body');
+
 	if (nav) {
 		// "Repository refresh" layout
+		const menu = select('.js-responsive-underlinenav-overflow ul')!;
+
 		nav.parentElement!.classList.add('rgh-has-more-dropdown');
-		select('.js-responsive-underlinenav-overflow ul')!.append(
+		menu.append(
 			<li className="dropdown-divider" role="separator"/>,
-			createDropdownItem('Compare', compareUrl),
-			pageDetect.isEnterprise() ? '' : createDropdownItem('Dependencies', dependenciesUrl),
-			createDropdownItem('Commits', commitsUrl),
-			createDropdownItem('Branches', `/${repoUrl}/branches`)
+			createDropdownItem('Compare', compareUrl, {}, DiffIcon),
+			pageDetect.isEnterprise() ? '' : createDropdownItem('Dependencies', dependenciesUrl, {}, PackageIcon),
+			createDropdownItem('Commits', commitsUrl, {}, HistoryIcon),
+			createDropdownItem('Branches', branchesUrl, {}, BranchIcon)
 		);
+
+		for (const tab of select.all<HTMLAnchorElement>(`
+			.js-responsive-underlinenav-item[data-tab-item="security-tab"],
+			.js-responsive-underlinenav-item[data-tab-item="insights-tab"]
+		`)) {
+			const menuItem = menu.querySelector(`[data-menu-item="${tab.dataset.tabItem!}"] > a`)!;
+			menuItem.parentElement!.removeAttribute('hidden');
+			tab.parentElement!.remove();
+
+			menuItem.replaceWith(
+				<a role="menuitem" className="rgh-reponav-more dropdown-item" href={tab.href}>
+					{[...tab.childNodes]}
+				</a>
+			);
+		}
+
+		await new Promise(resolve => {
+			window.addEventListener('load', () => {
+				setTimeout(resolve, 10);
+			});
+		});
+
+		for (const tab of select.all<HTMLAnchorElement>(
+			'.js-responsive-underlinenav-item[style="visibility: hidden;"]'
+		)) {
+			menu.querySelector(
+				`[data-menu-item="${tab.dataset.tabItem!}"] > a`
+			)!.replaceWith(
+				<a role="menuitem" className="rgh-reponav-more dropdown-item" href={tab.href}>
+					{[...tab.cloneNode(true).childNodes]}
+				</a>
+			);
+		}
+
```

Comment:
What's this new code for? Without any comments it's hard to understand what's going on

It's almost identical to the one above, perhaps it should be merged?

Reply:
>What's this new code for? 

https://github.com/sindresorhus/refined-github/pull/3482#discussion_r479651220



> perhaps it should be merged?

```javascript
const re = new Promise((r) => {
  window.addEventListener('load', r);
});

p('.js-responsive-underlinenav', {
  constructor: HTMLElement,
  subscribe: (e) => {
    const t = e.querySelectorAll('.js-responsive-underlinenav-item');
    return (
      (async function (e, t) {
        await re, bs(e, t);
      })(e, t),
      oe(window, 'resize', () => bs(e, t))
    );
  },
});
```
`style="visibility: hidden;` applied after complete page load

Reply:
My understand with that comment was that it would reduce code, not add more.

Reply:
> that comment was that it would reduce code

that comment was that it would [avoid replacing every link](https://github.com/sindresorhus/refined-github/pull/3482#discussion_r473076413).

Reply:
But it still does, now in 2 places: `menuItem.replaceWith(<a />)`

What I'd expect this code to do is:

1. Remove security + insights tabs
2. Unhide security + insights menu items

And potentially add icons, but that's unrelated

3. Clone the icon from the respective tab and `menuItem.append()` it